yt/yt/tests/integration/proxies/test_http_proxy.py

```diff
@@ -153,6 +153,20 @@ def check_role_updated():
         assert len(hosts) == 1
         assert not hosts[0]["banned"]
 
+    @authors("aleksandr.gaev")
+    def test_cluster_connection(self):
+        def get_cluster_connection(path):
+            return requests.get(self._get_proxy_address() + path)
+
+        url = "{}/api/v3/get?path=//sys/@cluster_connection".format(self._get_proxy_address())
+        cluster_connection = requests.get(url).content
+        assert len(cluster_connection) > 0
+
```

Comment:
Can we also do `get('//sys/@cluster_connection')` to check that new handler actually returns cluster connection and not something strange?